### PR TITLE
[feat|sde-backend] : added policy controller for policy management.

### PR DESCRIPTION
**Pull request by ChetanT-System**
### Added
- Added Policy Controller for policy management.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by sachinargade123 — APPROVED** [review]
LGTM

File: modules/sde-core/src/main/java/org/eclipse/tractusx/sde/core/utils/ListToStringConverter.java
Comment (adityagajbhiye9) [inline modules/sde-core/src/main/java/org/eclipse/tractusx/sde/core/utils/ListToStringConverter.java:1]: @ChetanT-System Please check copyright header, It should 2022-2024.

**Review by adityagajbhiye9 — CHANGES_REQUESTED** [review]
Please check comment.

**Review by adityagajbhiye9 — APPROVED** [review]
LGTM

**Comment by adityagajbhiye9** [comment]
@almadigabor please approve and merge PR.

**Comment by adityagajbhiye9** [comment]
@tomaszbarwicki  Please review and merge.

**Review by almadigabor — APPROVED** [review]

**Comment by adityagajbhiye9** [comment]
Please merge.

**Comment by almadigabor** [comment]
Hey! Please run the dash-tool again to check for dependencies changes. I ran it locally and seemed to be out of date. I would also suggest implementing the dependencies workflow to have an automated check on each PR like [here](https://github.com/eclipse-tractusx/portal-backend/blob/main/.github/workflows/dependencies.yaml).

**Comment by almadigabor** [comment]
[Here](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/actions/runs/8551116115/job/23429436330?pr=110#step:9:9) you can see why the dependencies workflow failed. And also an updated file is attached to the workflow run as artefact that you can download to review and commit [here](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/actions/runs/8551116115?pr=110).

**Comment by ChetanT-System** [comment]
@almadigabor  We have updated version in pom.xml file for dependencies check and also added workflow. Please review and merge again.

**Pull request merged**